### ISSUE TITLE
Application won't launch if several bluetooth peripheral devices.

### ISSUE THREAD

**Issue by Erstin** [picaso/parrot-zik-status#13]
My application ParrotStatus won't launch if I have multiple peripheral devices plugged on my Mac.
Today, I've seen this bug while I was connecting my bluetooth mouse.
At the moment when I connect the mouse, ParrotStatus quitted, and won't start.
When I disconnect the mouse, ParrotStatus launched correctly, and re-crashed after synchronizing my mouse.

**Comment by istrion** [comment]
Same issue here

**Comment by picaso** [comment]
Yes, this is a bug, unfortunately, I do not have the parrot headphone anymore. I am looking for someone to help maintain this repo.
